README.md

```diff
@@ -16,7 +16,7 @@ ngx-quill is the new angular (>=2) implementation of ngQuill.
 - include theme stylings: bubble.css, snow.css of quilljs in your index.html
 
 ### For standard webpack and tsc builds
-**With thew new version of angular CLI, you must serve your project with the AOT option.**
+**With the new version of angular CLI, you must serve your project with the AOT option.**
```

Comment:
could you add the correct version like with angular cli >v x.x? so it is clear for everyone, Thanks

Reply:
done